ru_RU/webapp/src/main/webapp/i18n/all_ru_RU.properties

```diff
@@ -0,0 +1,965 @@
+selected_profiles = Выбранные профили
+save_profile_changes = Сохранить изменения профилей
+
+#
+# page partials templates ( /templates/freemarker/page/partials )
+#
+copyright = авторские права
+all_rights_reserved = Все права защищены.
+terms_of_use = Условия использования
+
+site_name = имя сайта
+end_your_Session = Завершить ваш сеанс
+log_out = Выйти
+manage_site = Управлять сайтом
+site_admin = Администрирование
+more_details_about_site = Больше информации об этом сайте
+about = О системе
+contact_us = Связаться с нами
+send_feedback_questions = Отправьте свой отзыв или задайте вопрос
+visit_project_website = Посетите сайт национального проекта
+support = Поддержка
+view_content_index = Ознакомиться с кратким содержанием этого сайта
+index = Каталог
+search_form = Поисковая форма
+
+select_locale = выбраррррегиональные настройки
+language_selection_failed = В системе произошла ошибка. Выбор языка был отклонён.
+menu_item = пункт меню
+version = Версия
+
+#
+# widget templates ( /templates/freemarker/widgets )
+#
+individual_name = имя объекта
+vclassAlpha_not_implemented = vclassAlpha еще не внедрён.
+
+test_for_logged_in_users = Это пробный виджет для авторизованных пользователей.
+test_for_nonlogged_in_users = Это пробный виджет для неавторизованных пользователей.
+login_status = Состояние авторизации:
+
+alert_icon = Значок оповещения
+javascript_require_to_edit = Для редактирования необходимо разрешить JavaScript.
+javascript_instructions = инструкции по java script
+to_enable_javascript = Как разрешить JavaScript в браузере
+external_auth_name = имя сервиса внешней аутентификации
+external_login_text = Вход с помощью сервиса внешней аутентификации
+account = учётная запись
+change_password_to_login = Смените пароль для входа
+new_password_capitalized = Новый пароль
+confirm_password_capitalized = Подтвердите пароль
+already_logged_in = Вы уже вошли.
+
+#
+# lib templates ( /templates/freemarker/lib )
+#
+statistics = Статистика
+
+manage_list_of = Управление списком
+manage = управлять
+add = добавить
+entry = запись
+edit_entry = редактировать эту запись
+delete_entry = удалить эту запись
+click_to_view_larger = нажмите для просмотра увеличенного изображения
+photo = Фотография
+no_image = изображение отсутствует
+placeholder_image = стандартное изображение
+manage_labels = управлять метками
+manage_list_of_labels = управлять списками меток
+view_list_of_labels = просмотр списков меток
+view = вид
+add_label = добавить метк
+add_label_for_language = Язык
+unsupported_ie_version = Эта форма не поддерживается в версиях Internet Explorer ниже версии 8. Пожалуйста, обновите свой браузер или перейдите на другой браузер, например FireFox.
+
+#
+# edit templates ( /templates/freemarker/edit and edit/forms )
+#
+edit_capitalized = Редактирование
+add_capitalized = Добавить
+create_entry = Создать запись
+select_existing_collaborator = Выбрать существующего партнёра для {0}
+selected = Выбранное
+change_selection = Изменить выбранное
```

Comment:
This is in lowercase (with small letter at the beginning) in the English property file. I am not sure what is the context in which this property is used.

Reply:
It is an action to change selection in a form. It should start with upper case to look appropriate in Russian.